src/marqo/s2_inference/processing/text.py

```diff
@@ -30,6 +30,12 @@ def _splitting_functions(split_by: str, language: str='english') -> FunctionType
     except LookupError:
         nltk.download("punkt")
 
+    # Punkt_tab needs to be downloaded after NLTK 3.8 and later
```

Comment:
What is this doing?

Reply:
It is needed in the text split. Previously it was downloaded along with `"punkt"` but now we need to download it explicitly after the upgrade.

Reply:
An error will be raised if we do not do this downloading